src/test/common.js

```diff
@@ -49,25 +49,31 @@ let testId = uuid();
 let storageDir = config.get('settings.directory') + '-' + testId;
 
 beforeAll(async () => {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
```

Comment:
I'll check and see if its still needed. The root cause was that the nanomsg sockets needed to be closed or jest would wind up hanging because it detected "resources were still in use".